src/systemtest/resources/noDateRange/expected/reposense_testrepo-Alpha_master/commits.json

```diff
@@ -1,141 +1,134 @@
 {
-  "authorWeeklyIntervalContributions": {
+  "authorDailyContributionsMap": {
     "fakeAuthor": [
       {
         "insertions": 0,
         "deletions": 0,
-        "sinceDate": "2018-02-06",
```

Comment:
keep the sinceDate, untilDate, its will be easier to generate the github link to show just that period